plugin.cpp

```diff
@@ -309,6 +309,15 @@ static const char *default_config = QUOTE({
                 "order" : "40",
                 "group" : "OPC UA Security",
                 "validity" : " securityMode == \"Sign\" || securityMode == \"SignAndEncrypt\" "
+        },
+        "controlList" : {
+            "description" : "This specifies the permitted node IDs of OPC UA Variables that the plugin is authorized to write to.",
```

Comment:
Slightly awkward wording here. How about "A list of node IDs for the OPC UA variables that may be written via the plugin".